website/content/docs/intro/index.mdx

```diff
@@ -49,71 +41,50 @@ Review the video below to learn more about Consul from HashiCorp's co-founder Ar
-  be changed in real time instead of using complex network topologies
-  and static firewall rules.
-
-- **Multi Datacenter**: Consul supports multiple datacenters out of the box. This
-  means users of Consul do not have to worry about building additional layers of
-  abstraction to grow to multiple regions.
-
-Consul is designed to be friendly to both the DevOps community and
-application developers, making it perfect for modern, elastic infrastructures.
-
-## Basic Architecture of Consul
-
-Consul is a distributed, highly available system. This section will cover the
-basics, purposely omitting some unnecessary detail, so you can get a quick
-understanding of how Consul works. For more detail, please refer to the
-[in-depth architecture overview](/docs/architecture).
-
-Every node that provides services to Consul runs a _Consul agent_. Running
-an agent is not required for discovering other services or getting/setting
-key/value data. The agent is responsible for health checking the services
-on the node as well as the node itself.
-
-The agents talk to one or more _Consul servers_. The Consul servers are
-where data is stored and replicated. The servers themselves elect a leader.
-While Consul can function with one server, 3 to 5 is recommended to avoid
-failure scenarios leading to data loss. A cluster of Consul servers is recommended
-for each datacenter.
-
-The servers maintain a _catalog_, which is formed by aggregating information
-submitted by the agents. The catalog maintains the high-level view of the cluster,
-including which services are available, which nodes run those services, health
-information, and more. How agents and the catalog interact can be found
-[here](/docs/architecture/anti-entropy#catalog).
-
-Components of your infrastructure that need to discover other services
-or nodes can query any of the Consul servers _or_ any of the Consul agents.
-The agents forward queries to the servers automatically.
-
-Each datacenter runs a cluster of Consul servers. When a cross-datacenter
-service discovery or configuration request is made, the local Consul servers
-forward the request to the remote datacenter and return the result.
-
-## Next Steps
-
-Continue onwards with [HashiCorp Learn](https://learn.hashicorp.com/tutorials/consul/get-started-install)
-to learn more about Consul and how to get Consul up and running.
+### Automate service discovery
+
+Adopting a microservices architecture on cloud infrastructure is a critical step toward delivering value at scale, but knowing where healthy services are running on your networks in real time becomes a challenge. Consul automates service discovery by replacing service connections usually handled with load balancers with an identity-based service catalog. The service catalog is a centralized source of truth that you can query through Consul’s DNS server or API. The catalog always knows which services are available, which have been removed, and which services are healthy.
```

Comment:
I think we're useful beyond just adopting a microservices architecture on cloud infrastructure. In fact, I think the fact that we can help with that *and* also bring value to legacy monolithic applications, and services running on premise as well, is a definite value, because we can connect all of those together, and also help provide a path to move from on-premise to cloud, or from monolithic to microservices; by having some common networking tools which work across all of those we can reduce the friction to actually doing that moving.

Perhaps: "Organizations have seen a massive increase in the flexibility, scale, and speed at which they can build infrastructure, and they have taken advantage of those new capabilities to completely change how they architect and deploy applications. And at the same time, ideas like microservices-based architectures increase the speed and agility with which organizations can build those applications. But both of these lead to a vast increase in the amount of complexity, the sheer number of moving parts, and answering one of the most basic of questions: where are all of my services, and which instances of them are healthy and able to serve traffic, becomes a real challenge. By following common, simple patterns, as services are deployed, scaled, and retired, Consul's service catalog becomes a near real time, centralized source of truth for that information. The catalog always knows which services are available, and which of those are healthy. And because that catalog can be queried through a WebUI, a CLI, through DNS and through an API, it's easy for both people and your existing workflows to consume."